### DISCUSSION TITLE
Participation in Gsoc

### DISCUSSION THREAD

**Discussion by PrasanthChettri** [fastapi/fastapi#2828] [Questions] [answered]
Gsoc is an annual opensource even, where one can get some extra help for the repo, in exchange for some experience and mentorship, a lot of good devs participate in the event and all around it can be beneficial to the repo, if FastApi participates. What do you guys think ?

**Comment by tiangolo** [comment, marked as answer]
If you want to help, here are the best ways to do it 🤓  https://fastapi.tiangolo.com/help-fastapi/
